ledger/leios/leios.go

```diff
@@ -64,7 +64,8 @@ type LeiosEndorserBlock struct {
 	cbor.DecodeStoreCbor
 	cbor.StructAsArray
 	hash         *common.Blake2b256
-	TxReferences []common.TxReference
+	transactions []common.Transaction
+	TxReferences []common.TxReference `cbor:"0,keyasint"`
```

Comment:
This annotation doesn't make sense with `cbor.StructAsArray`, since it's only for a map